app/src/main/java/com/example/foody/recipe_search/presentation_mvvm/SearchViewModel.kt

```diff
@@ -69,8 +69,8 @@ class SearchViewModel @Inject constructor(
                 if (recipeList.isEmpty()) RecipeListState.Empty
                 else RecipeListState.Success(recipeList = recipeList)
             } catch (e: Exception) {
-                Log.e("RecipeSearchViewModel", e.message.orEmpty(), e)
```

Comment:
Logging is actually better. Try to import and use [Timber](https://github.com/JakeWharton/timber).
Doesn't have to be in this PR.

Reply:
Ok. But I would also like to hear why.

Reply:
Logging gets logged to logcat, and has levels, which logcats differentiates by color. VERBOSE (black), DEBUG (black), INFO (blue), ERROR (red), WTF (red). It is configurable as well. printing to console is just a simple print in black without any context. Logging also supports tagging, and logcat supports filtering logs by level, tag, search term, etc.
Timber is a logging library for Android which creates a "default" tag for you so you don't need to always for each log, and it is usually the class or the file from which it is logged.